backend/routes/WishWedRoutes.js

```diff
@@ -1,12 +1,24 @@
 const express = require("express");
+const sanitizeHtml = require("sanitize-html");
 const WishWedSchema = require("../models/WishWednesday");
 const { validate, primaryAdminValidator, errorHandler } = require("../util/RouteUtils");
 
 const router = express.Router();
 
 router.post("/add/", primaryAdminValidator, validate(["message"]), (req, res) => {
+  // need to sanitize html here
+  const cleanText = sanitizeHtml(req.body.message, {
+    allowedTags: sanitizeHtml.defaults.allowedTags.concat(["img"]),
+    allowedAttributes: {
+      img: ["src", "srcset", "alt", "title", "width", "height", "loading"],
```

Comment:
This and the next suggestion allow links to work.

```suggestion
      img: ["src", "srcset", "alt", "title", "width", "height", "loading"],
      a: ["href"],
```